examples/specs/interactive_bar_select_highlight.vl.json

```diff
@@ -0,0 +1,49 @@
+{
+  "$schema": "https://vega.github.io/schema/vega-lite/v3.json",
+  "description": "A bar chart with highlighting on hover and selecting on click. (Inspired by Tableau's interaction style.)",
+  "data": {
+    "values": [
+      {"a": "A","b": 28}, {"a": "B","b": 55}, {"a": "C","b": 43},
+      {"a": "D","b": 91}, {"a": "E","b": 81}, {"a": "F","b": 53},
+      {"a": "G","b": 19}, {"a": "H","b": 87}, {"a": "I","b": 52}
+    ]
+  },
+  "selection": {
+    "highlight": {"type": "single", "empty": "none", "on": "mouseover"},
+    "select": {"type": "multi"}
+  },
+  "mark": {
+    "type": "bar",
+    "fill": "#4C78A8",
+    "stroke": "black",
+    "cursor": "pointer"
```

Comment:
I think this should be the default. Let's not add it manually just here, okay?

Reply:
I disagree. I don't know when that default gonna land. You can undo that in a separate PR.

Reply:
Okay, please create an issue and link to this example then.

Reply:
lol you already created it a few months ago. https://github.com/vega/vega-lite/issues/4155

Reply:
Haha, shows how much I want it.